main.py

```diff
@@ -21,7 +21,7 @@ def main():
         sys.exit(errno.EPERM)
 
 
-
+    #TODO : move the RSS Link To Config (mDownloader.ini)
```

Comment:
This isn't a configuration, we always will go to this  URL.
But indeed we should move it to globals.py

Reply:
Fixed it :)